string robot_id
float64 transportation_rate
int32 max_n_trays
int32 n_empty_trays
int32 n_full_trays
int32 n_total_trays
int32 mode
bool loaded
